wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -20,7 +20,7 @@ Rule proposals as well as suggestions are discussed in the [Ranking Criteria Sub
 - **Column**: Vertical lines on the playfield where notes are placed. The editor uses `0|1|2|3|4|5|6|7|8` for timestamp labels. 
 - **Key mode**: The number of columns present in a difficulty.
 - **Key sounding**: Hit sounding using samples extremely similar or directly from the song in their respective places. This method of hit sounding usually provides low feedback to the player.
-- **Slider velocity**: The speed at which the notes/bar lines move on the playfield. Base slider velocity can be controlled in the timing panel and additional changes can be made through inherited (green) timing points.
+- **Slider velocity**: The speed at which the notes/bar lines move on the playfield. Base slider velocity (1.0x) is based on the user's set scroll speeds and additional changes can be made through inherited (green) timing points.
```

Comment:
```suggestion
**Slider velocity**(**Speed Variations**): The speed at which the notes/bar lines move on the playfield. Base slider velocity (1.0x) is based on the user's set scroll speeds and additional changes can be made through inherited (green) timing points.
```

Reply:
source : https://i.ppy.sh/e7ceb44562f4564affa4b964e19259f9d0e3cfa1/68747470733a2f2f7075752e73682f434831454c2f376666393062383835612e706e67
i dont know if it discussed on community before to not use Speed Variations instead of Slider velocity on community, need BNs or QATs who familiar with the community to determined.